openmetadata-ui/src/main/resources/ui/src/components/Settings/Users/CreateUser/CreateUser.component.tsx

```diff
@@ -123,7 +123,7 @@ const CreateUser = ({
       displayName: trim(displayName),
       roles: selectedRoles,
       teams: validTeam.length ? validTeam : undefined,
-      email: email,
+      email: lowerCase(email),
```

Comment:
Let's use `toLower` to get the whole word a lowercase as `lowerCase` Converts string, as space-separated words, to lowercase.